sys-block/di/di-4.54.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Configurable Disk Information Utility"
+DESCRIPTION="Disk Information utility with configurable output format"
+HOMEPAGE="https://diskinfo-di.sourceforge.io"
+COPYRIGHT="1994-2021 Brad Lanam"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/diskinfo-di/files/di-$portVersion.tar.gz/download"
+CHECKSUM_SHA256="f500fc1fe27586738e12d773dabc9af108fe4690d6bbc8220a9c3c3f774f8748"
+SOURCE_FILENAME="di-$portVersion.tar.gz"
+SOURCE_DIR="di-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	di$secondaryArchSuffix = $portVersion
+	cmd:di = $portVersion
+	cmd:mi = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make prefix=$prefix DATADIR=$dataDir CC=gcc
+}
+
+INSTALL()
+{
+	make prefix=$prefix DATADIR=$dataDir install
+}
```

Comment:
Missing MANDIR=$manDir for both (only INSTALL would probably do).
Atm manpage ends up in $dataDir/man... and doesn't show when invoked with "man di" in Terminal.

Reply:
Added.